### PR TITLE
KevS-SK1557-Create-SQL-EOS-EOL-for-WebHosting-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done.

Need proof by : 

- @tcpdumpfbacke for DE and EN subs
- @BrenKt for latin and PL subs

### CODE REVIEW

File: pages/web_cloud/web_hosting/sql_eos_eol/guide.fr-fr.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: 'Annonces de fin de vente/vie base de données SQL'
-excerpt: 'Annonces de fin de vente/vie base de données SQL'
+title: "Annonces de fin de vente/vie base de données SQL"
+excerpt: "Annonces de fin de vente/vie base de données SQL"
 updated: 2023-11-16
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/sql_eos_eol/guide.fr-fr.md]: dates should be all aligned

File: pages/web_cloud/web_hosting/sql_eos_eol/guide.en-gb.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: 'SQL Database EOS and EOL announcements'
-excerpt: 'SQL Database EOS and EOL announcements'
+title: "SQL Database EOS and EOL announcements"
+excerpt: "SQL Database EOS and EOL announcements"
 updated: 2023-11-16
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/sql_eos_eol/guide.en-gb.md]:
```suggestion
updated: 2024-05-27
```

File: pages/web_cloud/web_hosting/sql_eos_eol/guide.fr-fr.md (hunk 1 above)
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/sql_eos_eol/guide.fr-fr.md]:
```suggestion
updated: 2024-05-27
```

Comment (Kazarus4) [reply pages/web_cloud/web_hosting/sql_eos_eol/guide.fr-fr.md]: Done

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**